src/views/Home.vue

```diff
@@ -302,14 +303,23 @@ export default class Home extends Vue {
     this.dialog = true;
   }
 
+  roundToQuarter(time: Date) {
+    const hours = time.getHours()
+    const suffix = (hours >= 12) ? 'pm' : 'am'
+    const hoursRegular = (hours !== 0) ? hours % 12 : 12
+    const minutes = time.getMinutes()
+    const nearestQuarter = (minutes <= 30) ? 15 : 45
+
+    return `${hoursRegular}:${nearestQuarter}${suffix}`
```

Comment:
Use [Moment.js](https://momentjs.com/)

- https://momentjs.com/docs/#/plugins/round/

Reply:
Not sure if `moment-round` can round to the nearest `:15`/`:45`.
This is my rounding code from `index.ts` (that does the same thing but returns a moment instead of a string):
```
function roundDate(d: Date) {
  const date = moment(d);
  date.millisecond(Math.floor(date.millisecond() / 1000) * 1000);
  date.second(Math.floor(date.second() / 60) * 60);
  if (round30) {
    date.minute(Math.round((date.minute() + 15) / 30) * 30 - 15);
  } else {
    date.minute(Math.round(date.minute() / 15) * 15);
  }
  return date;
}
```
If you want to return a string, you can `return date.format("h:mma")` instead.